iati/core/rulesets.py

```diff
@@ -142,12 +144,7 @@ def _normalize_xpath(self, path):
         return self.xpath_base + '/' + path
 
     def _normalize_xpaths(self):
-        """Normalize xpaths by combining them with `xpath_base`.
-
-        Todo:
-            Discuss appropriate edge cases and what is going on here.
-
-        """
+        """Normalize xpaths by combining them with `xpath_base`."""
```

Comment:
Based on earlier confusion, probably worth expanding the docstring to detail why this may want overriding in child classes (when there is no `paths` attribute and such)